python/paddle/cost_model/cost_model.py

```diff
@@ -53,17 +54,33 @@ def profile_measure(self,
         exe.run(startup_program)
         paddle.fluid.profiler.start_profiler("All")
         exe.run(main_program, feed={"X": x}, fetch_list=[])
-        # core.CostModel.ProfileMeasure(main_program, device)
-        print("core:<<<<<<<")
 
         cost_model = core.CostModel()
         cost_data = cost_model.ProfileMeasure(device)
-        # cost_list = self.stop_cost_model()
-        # return cost_list
 
+    def static_cost_data(self):
+        static_cost_data_path = os.path.join(
+            os.path.dirname(__file__), "static_op_benchmark.json")
```

Comment:
A little suggestion, "static_op_benchmark.json" can be set as a constant at head of file. In the code, we may not find it easily. However, you can do it in next PR

Reply:
Cool, will do it in the next PR.